spec/lib/mas/firm_result_spec.rb

```diff
@@ -128,6 +128,17 @@
       expect(subject.adviser_qualification_ids).to eq([3])
     end
 
+    describe '#advisers' do
+      it 'returns an array containing the advisers' do
+        expect(subject.advisers).to be_an(Array)
+        expect(subject.advisers.length).to eq(1)
+      end
+
+      it 'returns AdviserResult objects' do
+        expect(subject.advisers.first).to be_an(AdviserResult)
+      end
+    end
```

Comment:
for future reference we have rspec collection matchers in this codebase. not worth amending though unless you really really want to